source/exe/win32/service_base.h

```diff
@@ -9,6 +9,8 @@
 
 #include "envoy/event/signal.h"
 
+#include "common/common/event_logger_impl.h"
```

Comment:
nit: I'm not sure I would bother with the strip prefix on this include given this is really win32 specific and included from a win32 specific location?